pkg/cmd/api/main.go

```diff
@@ -39,7 +40,7 @@ func Main(cfg *config.Config) {
 	logger.Printf("Api service start http://%s:%d\n", cfg.Api.Host, cfg.Api.Port)
 
 	if err := run(cfg); err != nil {
-		log.Fatal(err)
+		log.Fatalf("%+v", err)
```

Comment:
Whether it should be logger?

Reply:
yes, we should use logger instead of std log

Reply:
if all logger.Fatalf should be like this, can we change looger.Fatalf to 
```
func Fatalf(format string, args ...interface{}) {
	if len(args) > 0 {
		args[0] = errors.WithStack(args[0])
	}
	logger.Fatalf(format, args...)
	// Make sure fatal logs will exit.
	os.Exit(1)
}
```

Reply:
it is a wrong practice, `errors.WithStack` will include the `logger.Fatalf` call frames.

Reply:
ok, got it